tests/assets/spa/errors/discarded-interaction.html

```diff
@@ -19,7 +19,7 @@
     document.addEventListener('click', function () {
       setTimeout(function foo () {
         (function bar () {
-          fetch('/json')
+          if (typeof fetch !== 'undefined') fetch('/json') // because IE sucks
```

Comment:
Why not just use XHR here?

Reply:
Valid, I think I did it bc fetch is easier with 1 line instead of 4 or 5. 😆